### ISSUE TITLE
Newest C tensorflow 2.9 is built with macOS 11 (BigSur) 

### ISSUE THREAD

**Issue by ekrich** [ekrich/stensorflow#26]
Although it does compile and run with macOS 10.15, it seems that it is build with at least 12.1. Github actions still supports 10.15 so I think I should use the oldest version that works but a newer version without the linking error is probably preferable.
```
[info] Compiling to native code (6618 ms)
[error] ld: warning: dylib (/usr/local/lib/libtensorflow.so) was built for newer macOS version (11.1) than being linked (10.15)
```

**Comment by ekrich** [comment]
Same with `2.10` perhaps CI should upgrade.

**Comment by ekrich** [comment]
CI is now using macOS 11 and tensorflow is now at 2.11. Closed by #41
